Examples/Examples/FormViewExampleView.swift

```diff
@@ -133,7 +134,12 @@ private extension URL {
 }
 
 /// The model class for the form example view
-@MainActor class Model: ObservableObject {
+@MainActor
+class Model: ObservableObject {
+    /// A Boolean value indicating whether or not to display all validation errors in the form, irrespective of
+    /// whether a given field has received user-focus.
+    @Published var validationIsForced = false
```

Comment:
If this is not intended to be set from outside of the model, then it should be `private(set)`. However, this is set inside of `submitChanges()` before any other work is done. That method is called by the view and that is the only place that this property is used by the model. This could be a state property on the view, the view could update the value when the Submit button is tapped/pressed, and then the model wouldn't even need to know about it. I see zero reason why this should be a property of the model.